boards/FSM/Inc/Io/Io_MSP3002K5P3N1.h

```diff
@@ -1,5 +1,12 @@
 #pragma once
 
+/**
+ * Get the brake actuation status from the MSP-300-2K5-P-3-N-1 brake pressure
+ * sensor
+ * @return True if the brake is actuated, false if the brake is not actuated
+ */
+bool Io_MSP3002K5P3N1_BrakeIsActuated(void);
```

Comment:
nit; function rename
```suggestion
bool Io_MSP3002K5P3N1_IsBrakeActuated(void);
```